client/lib/posts/actions.js

```diff
@@ -162,11 +164,10 @@ PostActions = {
 		} );
 	},
 
-	autosave: function( callback ) {
+	autosave: function( callback, site ) {
```

Comment:
I understand the intent of extend these methods, but we're usually placing the `site` (or `siteId`) as the first parameter. E.g.
```es6
autosave: function( site, callback ) { ... }
```

Now, since we're modifying both the functions definitions and all their uses, I think it would be more consistent to place the site first, on this and pretty much all other changed functions.

Reply:
I think at the time the decision to just add a parameter at the end was related with the way I was going to divide the PR's, but as you referred we are also changing the occurrences in this approach that I end up using. So I will try to make this change and the others relate with redundant site, siteId